src/modules/flow/netctl/netctl.json

```diff
@@ -0,0 +1,139 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "netctl",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "Apache-2.0",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "netctl",
+      "description": "Send list of all network services attached to board.",
+      "methods": {
+        "open": "open_list_services",
+        "close": "close_list_services"
+      },
+      "name": "netctl/list-services",
+      "out_ports": [
+        {
+          "data_type": "string",
+          "description": "Name of network services supported by system.",
+          "name": "OUT"
+        }
+      ],
+      "private_data_type": "list_service_data"
+    },
+    {
+      "category": "netctl",
+      "description": "Connect or disconnect to one available network access point.",
+      "in_ports": [
+        {
+          "data_type": "any",
+          "description": "Trigger to connect to network service.",
+          "methods": {
+            "process": "connect_service_name"
+          },
+          "name": "CONNECT"
+        },
+        {
+          "data_type": "any",
+          "description": "Trigger to disconnect to network service.",
+          "methods": {
+            "process": "disconnect_service_name"
+          },
+          "name": "DISCONNECT"
+        }
```

Comment:
you need two more input ports:
 - `TRIGGER`: Triggers emission of current service state.
 - `NAME`: set the service name to use (allows changing what is an option in runtime)

See example at https://github.com/solettaproject/soletta/blob/master/src/modules/flow/platform/platform.json#L576 we should make this node similar to that one (except ports START/STOP being called CONNECT/DISCONNECT).